docs/course_materials/exercises/01_Linux/check_class_exercise_1.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import os.path
+from os import path
+from pathlib import Path
+
+home = str(Path.home())
+exercise_commands = ['mkdir', 'tree']
+
+def get_student_commands():
+    '''
+    Checks stutent history file for key commands.
+    If the history is longer than 1000 commands, checks only 1000 last commands.
+    '''
+    my_commands = []
+    hist_path = home + '/.bash_history'
+    with open(hist_path, 'r') as rf:
+        for line in rf:
+            my_commands.append(line[:-1])
+    if len(my_commands) > 1000:
+        my_commands = my_commands[-1000:]
+    return my_commands
+
+def used_right_commands(my_commands):
+    '''
+    Checks if the student used the correct commands. 
+    ''' 
+    for command in exercise_commands:
+        return any(command in performed_command for performed_command in my_commands)
+    return True
+
+def created_directories():
+    return path.exists(home + '/test/tmp')
+
+def main():
+    '''
+    Running the tests and prompting the student according to his work.
+    '''
+    test_result = ''
+    student_commands = get_student_commands()
+    if not used_right_commands(student_commands):
+        test_result += ("It seems you missed some commands. "
+                    "Make sure you have used 'mkdir' for creating the directories "
```

Comment:
I think we can generate this message to write exactly which command wasn't run properly.